### PR TITLE
Update godaddy.md

### PR DESCRIPTION
**Pull request by c0nfigurati0n**
Updated the DKIM selection and corrected some spelling mistakes.

### CODE REVIEW

File: docs/custom-domain/registrars/godaddy/godaddy.md [hunk 1]
```diff
@@ -94,7 +104,7 @@ If it hasn't, then double check the setup and fix any mistakes.
 
 ## (Optional) Adding Domain-based Message Authentication Reporting & Conformance to GoDaddy
 
-Once you have set up the SPF and DKIM policies for your domain, setting up a DMARC (Domain-based Message Authentication Reporting & Conformance) policy is also recommended to even further reduce the chance of your emails ending up in the recipient's Spam Folder.
+Once you have setup the SPF and DKIM policies for your domain, setting up a DMARC (Domain-based Message Authentication Reporting & Conformance) policy is also recommended to even further reduce the chance of your emails ending up in the recipient's spam folder.
```
Comment (nguyenkims) [inline docs/custom-domain/registrars/godaddy/godaddy.md:107]: same here, "set up" feels more correct

**Review by nguyenkims — COMMENTED** [review]
Thanks a lot for making this PR! I just added some comments.

File: docs/custom-domain/registrars/godaddy/godaddy.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ In GoDaddy, press the "ADD" button, then select the following record Type, Name,
 
 ![](./spf-record.png)
 
-Now, after having done that, go back to the SimpleLogin DNS setup page and press on "Verify" to see if the SPF selection has been verified or not.
+Now, after having done that, go back to the SimpleLogin DNS setup page and press on "Verify" to see if the SPF selection has been verified yet or not.
```
[the remark this answers is not in the document]
Comment (c0nfigurati0n) [reply docs/custom-domain/registrars/godaddy/godaddy.md:67]: I've seen many people use the word yet this way. So how is this "wierd"?

File: docs/custom-domain/registrars/godaddy/godaddy.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Head to the SimpleLogin [domain page](https://app.simplelogin.io/dashboard/custo
 
 ![](./new-domain.png)
 
-You'll then be redirected to the SimpleLogin DNS setup page that has a guide on how to set up different DNS records for your domain.
+You'll then be redirected to the SimpleLogin DNS setup page that has a guide on how to setup different DNS records for your domain.
```
[the remark this answers is not in the document]
Comment (c0nfigurati0n) [reply docs/custom-domain/registrars/godaddy/godaddy.md:9]: Yeah, to my understanding it is. But i tend to make mistakes just like any human. So thank you for poining it out.

**Comment by c0nfigurati0n** [comment]
Seems good to me? You can marge it if you want?

**Review by nguyenkims — APPROVED** [review]

**Pull request merged**